ftdetect/sml.vim

```diff
@@ -1,2 +1,5 @@
 " SML signature files
-au BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup VimBetterSMLFtDetect
+  autocmd!
+  autocmd BufRead,BufNewFile *.sig setlocal filetype=sml
+augroup END
```

Comment:
In my experience this actually depends on the plugin manager you're using.  But, if the manual says you shouldn't put it there...